test/integration/services/patient-quick-search.js

```diff
@@ -1,10 +1,12 @@
 import _ from 'underscore';
-import patientFixture from 'fixtures/collections/patients.json';
-import { getRelationship, getIncluded } from 'helpers/json-api';
+
+import { getResource, getRelationship, getIncluded } from 'helpers/json-api';
+
+import fxPatients from 'fixtures/collections/patients';
 
 context('Patient Quick Search', function() {
   beforeEach(function() {
-    const patients = _.map(_.sample(patientFixture, 10), (patient, index) => {
+    const patients = _.map(getResource(_.sample(fxPatients, 10), 'patients'), (patient, index) => {
```

Comment:
I don't think this works.  we just need to clone it here.. it's being jsonapi'd in the `getRelationship`, the `getInclude` and manually in the map below.